feature-libs/product-configurator/rulebased/components/restart-dialog/configurator-restart-dialog.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="cx-modal-container">
+  <ng-container *ngIf="dialogData$ | async as dialogData">
+    <ng-container *ngIf="product$ | async as product">
+      <div
+        class="cx-modal-content"
+        [cxFocus]="focusConfig"
+        (esc)="backToPDP(product)"
+      >
+        <div class="cx-dialog-header modal-header">
+          <div class="cx-dialog-title modal-title">
+            {{ 'configurator.restartDialog.title' | cxTranslate }}
+          </div>
+          <button
+            title="{{ 'configurator.a11y.closeRestartDialog' | cxTranslate }}"
+            type="button"
+            class="close"
+            (click)="backToPDP(product)"
+          >
+            <span aria-hidden="true">
+              <cx-icon [type]="iconTypes.CLOSE"></cx-icon>
+            </span>
+          </button>
+        </div>
+
+        <div class="cx-dialog-body modal-body">
+          <div
+            class="cx-configurator-dialog-description"
+            id="cx-configurator-restart-dialog-description"
+          >
+            {{ 'configurator.restartDialog.description' | cxTranslate }}
+          </div>
+          <button
+            class="btn btn-primary btn-block cx-configurator-dialog-resume"
```

Comment:
Please, remove `cx-configurator-dialog-resume` class. `btn-primary` class is enough to style this button accordingly.
The goal is to hold HTML tree as simple as possible.